schemas/v3.0/schema.yaml

```diff
@@ -0,0 +1,1500 @@
+        default: false
+      content:
+        type: object
+        additionalProperties:
+          $ref: '#/definitions/MediaType'
+        minProperties: 1
+        maxProperties: 1
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+
+  RequestBody:
+    type: object
+    required:
+      - content
+    properties:
+      description:
+        type: string
+      content:
+        type: object
+        additionalProperties:
+          $ref: '#/definitions/MediaType'
+      required:
+        type: boolean
+        default: false
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+
+  SecurityScheme:
+    oneOf:
+      - $ref: '#/definitions/APIKeySecurityScheme'
+      - $ref: '#/definitions/HTTPSecurityScheme'
+      - $ref: '#/definitions/OAuth2SecurityScheme'
+      - $ref: '#/definitions/OpenIdConnectSecurityScheme'
+  
+  APIKeySecurityScheme:
+    type: object
+    required:
+      - type
+      - name
+      - in
+    properties:
+      type:
+        type: string
+        enum:
+          - apiKey      
+      name:
+        type: string
+      in:
+        type: string
+        enum:
+          - header
+          - query
+          - cookie
+      description:
+        type: string
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+  
+  HTTPSecurityScheme:
+    oneOf:
+      - $ref: '#/definitions/NonBearerHTTPSecurityScheme'
+      - $ref: '#/definitions/BearerHTTPSecurityScheme'
+
+  NonBearerHTTPSecurityScheme:
+    not:
+      type: object
+      properties:
+        scheme:
+          type: string
+          enum:
+            - bearer
+    type: object
+    required:
+      - scheme
+      - type
+    properties:
+      scheme:
+        type: string
+      description:
+        type: string
+      type:
+        type: string
+        enum:
+          - http
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+
+  BearerHTTPSecurityScheme:
+    type: object
+    required:
+      - type
+      - scheme
+    properties:
+      scheme:
+        type: string
+        enum:
+          - bearer
+      bearerFormat:
+        type: string
+      type:
+        type: string
+        enum:
+          - http
+      description:
+        type: string
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+
+  OAuth2SecurityScheme:
+    type: object
+    required:
+      - type
+      - flows
+    properties:
+      type:
+        type: string
+        enum:
+          - oauth2
+      flows:
+        $ref: '#/definitions/OAuthFlows'
+      description:
+        type: string
+    patternProperties:
+      '^x-': {}
+    additionalProperties: false
+
+  OpenIdConnectSecurityScheme:
+    type: object
+    required:
+      - type
+      - openIdConnectUrl
+    properties:
+      type:
+        type: string
+        enum:
+          - openIdConnect      
+      openIdConnectUrl:
+        type: string
+        format: url
```

Comment:
@jblazek spotted that this either should be `format: uriref` or there is an omission in the spec requiring an absolute URL.